core/src/main/java/org/radargun/stages/test/TestStage.java

```diff
@@ -73,6 +75,9 @@
    @Property(doc = "Enable this property in order to show the difference between latency and service.")
    protected boolean reportLatencyAsServiceTime;
 
+   @Property(doc = "TCP congestion control to auto-detect concurrency limits for services in order to achieve optimal throughput with optimal latency.")
+   protected boolean clientLimiter;
```

Comment:
Sorry, I forgot to ask this before. Can you explain how this works and doesn't introduce some new flavor of coordinated omission?

Reply:
Removed